README.md

```diff
@@ -90,6 +91,17 @@ Options can be combined:
 ./build.sh run --scale 7.0 --seed 42 'abstract art'
 ```
 
+On systems with <8GB of GPU RAM, you can try mixing and matching options:
+
+* Make images smaller than 512x512 using `--W` and `--H`
```

Comment:
```suggestion
* Make images smaller than 512x512 using `--W` and `--H`, which proportionately reduces memory usage and increases generation speed
```

Reply:
I'm not sure if the change is precisely **proportional**, so I omitted that part of the description.